repo/local_config.go

```diff
@@ -12,10 +12,12 @@ import (
 
 // LocalConfig is a configuration of Kopia stored in a configuration file.
 type LocalConfig struct {
-	Storage  blob.ConnectionInfo    `json:"storage"`
-	Caching  content.CachingOptions `json:"caching"`
-	Hostname string                 `json:"hostname"`
-	Username string                 `json:"username"`
+	APIServer *APIServerInfo `json:"apiServer"`
```

Comment:
IIUC: a client can either directly use blob storage or go through the APIServer. Is that correct?

It makes sense to explicitly add a comment either to the struct or the fields about the intended usage.

Reply:
done.